neuroinformatics/hbp_prov/hbp-prov-schema-v3.1.json

```diff
@@ -519,57 +525,26 @@
             "required": [
               "bounding_box"
             ],
-            "oneOf": [
-              {
-                "properties": {
-                  "atlas_template": {
-                    "description": "The description of the reference atlas to which the dataset was aligned. This value should be provided when the reference atlas is not registered as a dataset. However, it can be provided when the current dataset is itself a reference atlas for description purpose.",
-                    "$ref": "#/definitions/onto_term"
-                  }
-                },
-                "required": [
-                  "atlas_template"
-                ]
-              },
-              {
-                "properties": {
-                  "parent_space": {
-                    "type": "string",
-                    "description": "The reference of the parent space from which the bounding box was extracted. The parent space should be registered as a dataset.",
-                    "pattern": "^[a-f0-9]{8}-[a-f0-9]{4}-[a-f0-9]{4}-[a-f0-9]{4}-[a-f0-9]{12}$"
-                  }
-                },
-                "required": [
-                  "parent_space"
-                ]
-              }
-            ],
             "properties": {
+              "atlas_template": {
```

Comment:
The description should mention that 'atlas_template' and 'parent_space' are exclusive properties. '...IMPORTANT: atlas_template or parent_space must be specified...' is not enough to express that. I would rather say 'IMPORTANT: atlas_template and parent_space properties are exclusive properties' and put that on dataset[*].atlas_location.description.

For me removing the 'one of' clause means that the validation of this exclusivity is not manage by the schema but hbp-prov validator should check that.